datasets/za/pmg_legislators/za_pmg_legislators.yaml

```
@@ -0,0 +1,35 @@
+title: South Africa National and Provincial Legislature and Executives
+entry_point: crawler.py
+prefix: za-pmg-leg
+coverage:
+  frequency: daily
+  start: 2024-08-08
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: > 
+ Current members of the National Assembly, National Council of Provinces,
+ and provincial legislatures and executives in South Africa.
+description: |
+    This dataset includes information about current members of South Africa's
+    National Assembly, National Council of Provinces, provincial legislatures,
+    and provincial executives. It covers various positions in the National
+    Legislature and Executive branch, including Ministers and members of the
+    Executive Committee.
```

Executive Committees are provincial bodies

```suggestion
    Legislature and Executive branch.
```